docker-compose.yaml

```diff
@@ -86,6 +86,7 @@ services:
       OKTA_CLIENT_ID: "${OKTA_CLIENT_ID}"
       OKTA_CLIENT_SECRET: "${OKTA_CLIENT_SECRET}"
       ENV_STATE: "${ENV_STATE}"
+      ID_MATI_URL: "${ID_MATI_URL}"
```

Comment:
@sweng66 I don't know which services need this value, but I'll defer to you and @valearna

Reply:
API and crontab all need ID_MATI_URL for getting the new curie.